<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_21">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_13">
       <layout class="QGridLayout" name="gridLayout_16">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_20">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <spacer name="verticalSpacer_13">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QLineEdit" name="username_login">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string comment="username" extracomment="username"/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="password_login">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <strikeout>false</strikeout>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">password_login[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string comment="password" extracomment="password"/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_34">
                <item>
                 <widget class="QPushButton" name="login_button">
                  <property name="minimumSize">
                   <size>
                    <width>98</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Login</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="register_button">
                  <property name="minimumSize">
                   <size>
                    <width>97</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Register</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="save_login">
                  <property name="text">
                   <string>Save Login Info</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_21">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_14">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_14">
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_22">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <spacer name="verticalSpacer_15">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_35">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Username:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="username_regForm">
                  <property name="placeholderText">
                   <string comment="Username" extracomment="Username"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_36">
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>First Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="first_name_regForm">
                  <property name="placeholderText">
                   <string comment="first_name" extracomment="first_name"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_37">
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>Last Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="last_name_regForm">
                  <property name="placeholderText">
                   <string comment="last name" extracomment="last name"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="passwords">
                  <property name="text">
                   <string>Password:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="password_regForm">
                  <property name="placeholderText">
                   <string comment="password" extracomment="password"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_39">
                <item>
                 <widget class="QLabel" name="label_28">
                  <property name="text">
                   <string>Confirm password:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="confirm_password_regForm">
                  <property name="placeholderText">
                   <string comment="reenter password" extracomment="renter password"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_40">
                <item>
                 <widget class="QLabel" name="label_29">
                  <property name="text">
                   <string>Email:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="email_regForm">
                  <property name="placeholderText">
                   <string comment="email" extracomment="email"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_41">
                <item>
                 <widget class="QLabel" name="label_30">
                  <property name="text">
                   <string>Gender:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="male_regForm_4">
                  <property name="text">
                   <string>male</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="female_regForm">
                  <property name="text">
                   <string>female</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_31">
                  <property name="text">
                   <string>birthdate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateodB"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_43">
                <item>
                 <widget class="QPushButton" name="Back">
                  <property name="text">
                   <string>Back</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_23">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="registration_regForm_4">
                  <property name="text">
                   <string>Register</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_16">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_24">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_15">
       <layout class="QGridLayout" name="gridLayout_20">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item>
           <widget class="QPushButton" name="Count_Btn">
            <property name="text">
             <string>Count</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_16">
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>100</y>
          <width>561</width>
          <height>201</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>31</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ragastered</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
